docs/products/opensearch/concepts/access_control.rst

```diff
@@ -1,23 +1,41 @@
 Access control
 ==============
 
-Aiven for OpenSearch® supports index-level access control lists (ACL) to control permissions. This approach allows you to limit the operations that are available to specific connections and to restrict access to certain data sets, which improves the security of your data.
+Aiven for OpenSearch® supports index-level access control lists (ACL) to control permissions.
+This approach allows you to limit the operations that are available to specific service users and
+restrict access to certain data sets.
+
+.. note::
+   When access control is initially enabled for the service, a set of rules that allows
+   full unlimited access to existing service users is created automatically.
+
+Rules are defined separately for each user as ``pattern/permission`` combinations. The ``pattern`` defines the indices that the permission applies to.
+Patterns are glob-style, where ``*`` matches any number of characters (including none) and ``?`` matches any character.
 
 You can grant the following permissions:
 
-* ``deny``: no access
-* ``admin``: full access to APIs and documents 
+* ``deny``: explicitly deny access
+* ``admin``: unlimited access to the index
 * ``readwrite``: full access to documents
 * ``read``: allow only searching and retrieving documents
 * ``write``: allow updating, adding, and deleting documents
 
-  .. note::
-     Write permission allows the service user to create new indexes that match the pattern, but it does not allow deletion of those indexes.
+The permission also implies which *index APIs* the service user can access:
```

Comment:
Do we have a link to these Index APIs with more content?

Reply:
I found list of REST Index API in the OpenSearch documentation https://opensearch.org/docs/latest/opensearch/rest-api/index-apis/index/ , however, `search`, for example is not part of it, and the list includes only API directly related to indices.  So it doesn't look right.  

@mteiste, is there a reference for these "Index APIs" that we can use? Or perhaps to align with OpenSearch docs we shouldn't call them` Index APIs`? I might be missing something here.

Reply:
There is documentation in OpenSearch with Index API for REST https://opensearch.org/docs/latest/opensearch/rest-api/index-apis/index/ Even though it is for REST and our ACLs are for any communication mechanism, I think the link can be added. @mteiste correct me if I'm wrong.